dask/dataframe/io/tests/test_sql.py

```diff
@@ -11,7 +11,7 @@
 pytest.importorskip('sqlite3')
 
 
-data = """
+data = u"""
```

Comment:
I recommend adding `from __future__ import unicode_literals` to the top of this page instead of using the Python 2 `u'abc'` syntax